src/styles/header.scss

```
@@ -0,0 +1,68 @@
+:root {
+  --header-hight: 60px;
+  --active-bar-hight: 4px;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
```

move it to `reset.scss` and don't use `box-sizing: border-box;` on `body` tag.
if u want to use this property to entire project - use `*` tag
```suggestion
* {
  box-sizing: border-box;
}
}

body {
  font-family: Roboto, Helvetica, sans-serif;
  margin: 0;
}
```